### PR TITLE
Change REST API query to handle hyphen, WebLogic Logging Exporter

### PR DESCRIPTION
**Pull request by hzhao-github**
1. Issue: after upgrading to ELK Stack v7, the query using REST API that contains "managed-server" doesn't work properly and return all data from Elasticsearch repository that contain the word "managed". e.g. a REST API query requesting all data that contains "serverName = managed-server1:
curl --connect-timeout 5 --max-time 5 -X GET http://elasticsearch.default.svc.cluster.local:9200/wls/_count?q=serverName:managed-server1

It actually returns all data that contain "serverName = managed"

2. The issue appears since ELK Stack version 7.0.0

3. Cause: per the doc https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html, hyphen/dash is a reserved characters since ELK Stack 7 and later 

4. There are several ways to escape the reserved char like hyphen. This test chooses to use the query below

curl --connect-timeout 5 --max-time 5 -X GET  "http://elasticsearch.default.svc.cluster.local:9200/wls/doc/_count?pretty" -H 'Content-Type: application/json' -d '
{
"query" : {
"query_string" :
{ "query" : "managed-server1", 
   "fields" : ["serverName"], 
   "default_operator": "AND" }
}
}'

Jenkins job passed:

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1963/

and ItElsaticTest passed by itself
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1810/

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java [hunk 1]
```diff
@@ -302,9 +302,14 @@ public void testWlsLoggingExporter() throws Exception {
     // curl -X GET http://elasticsearch.default.svc.cluster.local:9200/wls/_count?q=serverName:managed-server1
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java:302]: Remove commented code

Comment (hzhao-github) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java]: done

**Review by anpanigr — APPROVED** [review]
LGTM

**Review by sankarpn — APPROVED** [review]
LGTM

**Pull request merged**